src/Pages/Cart/Cart.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import CartList from './Components/CartList/CartList';
+import './Cart.scss';
+
+class Cart extends Component {
+  constructor() {
+    super();
+    this.state = {
+      cartList: {},
+      totalQuantity: 0,
+      totalPrice: 0,
+    };
+  }
+
+  componentDidMount() {
+    fetch(`http://localhost:3000/data/Carts.json`)
+      .then(res => res.json())
+      .then(res => {
+        this.setState({ cartList: res });
```

Comment:
```suggestion
        this.setState({ cartList: res.Carts });
```
- 9번째 줄 리뷰와 연결되는 내용입니다. 이와 같이 필요한 데이터에 올바르게 접근해서 사용해주시기 바랍니다!

Reply:
수정하였습니다~~ 감사합니다 준식님!!☺️💖